my history/style.css

```
@@ -16,30 +16,51 @@ body {
   margin: 0 auto;
 }
 
+.flex {
+  display: flex;
+}
+
+button {
+  padding: 10px 40px;
+  font-size: 18px;
+  font-weight: 600;
+  background-color: rgb(196, 124, 189);
+  border: 0;
+  border-radius: 30px;
+  cursor: pointer;
+  transition: 0.2s;
+}
+
+button:hover,
+form .btn-enviar input:hover {
+  box-shadow: 0px 0px 8px 0px rgb(196, 124, 189);
```

Dica: @francieli-alvarenga você pode omitir a unidade `px` quando for setar algo como 0

```suggestion
  box-shadow: 0 0 8px 0 rgb(196, 124, 189);
```